task5/src/main/java/com/clouway/adapter/http/LoginFilter.java

```diff
@@ -35,6 +35,7 @@ public void destroy() {
   public void doFilter(ServletRequest req, ServletResponse resp, FilterChain chain) throws ServletException, IOException {
     HttpServletRequest request = (HttpServletRequest) req;
     HttpServletResponse response = (HttpServletResponse) resp;
+    sessionRepository.cleanExpired();
```

Comment:
`clenaExpired()` is called here and in `LoginController` ... do you need it in both places. Do you call it in other places?